modules/serverless-access-custom-domain.adoc

```diff
@@ -12,15 +12,23 @@
 
 [source,terminal]
 ----
-$ curl -H "Host: custom-ksvc-domain.example.com" http://<ip_address>
+$ curl -H "Host: custom-ksvc-domain.example.com" \
+    http://<ingress-router-IP-or-hostname>
 ----
 
-+
```

Comment:
I'd restore these lines and make the description:
> where `<ingress-router-IP-or-hostname> is the IP address or host name that the {product-title} ingress router is exposed to.

Reply:
Hope it's how you like to have it.